about.html

```diff
@@ -1,306 +1,295 @@
-						even elevate the human condition as we know it today?
-					</p>
-					<p>
-						Through these experiences, I realize how the past, present, and
-						future are part of an overlapping continuum of human experience -
-						like deep roots coming out of a mountain.
-					</p>
-
-					<p class="about-signature">ITLCT Co-Founder, Natasha Rabsatt</p>
-				</article>
-			</div>
-		</main>
-
-		<!-- Footer -->
-		<footer>
-			<div class="wrapper">
-				<!-- 3 Column Layout Container -->
-				<div class="column-container">
-					<!-- Company Column -->
-					<div class="company">
-						<!-- Footer Logo -->
-						<a href="/index.html" class="img-link">
-							<img
-								src="/assets/logos/itlct-logo.svg"
-								alt="If These Lands Could Talk Logo"
-							/>
-						</a>
+				<div class="flex-container">
+					<!-- ITLCT Logo -->
+					<a
+						href="/index.html"
+						class="img-link"
+						aria-label="If These Lands Could Talk Home Page Link">
+						<img
+							src="/assets/logos/itlct-logo.svg"
+							alt="If These Lands Could Talk Logo" />
+					</a>
+					<!-- Navigation Menu -->
+					<div class="mobile-menu-container">
+						<!-- mobile menu button -->
 						<button
-							class="land-ack-btn"
-							data-role="open-modal"
-							data-type="land-ack-modal"
-						>
-							Land Acknowledgement
+							id="mobile-menu"
+							class="mobile-menu"
+							aria-label="toggle mobile menu on">
+							<!-- Hamburger Icon -->
+							<img
+								src="/assets/icons/global/menu-icon.svg"
+								alt="mobile menu icon"
+								class="menu-icon" />
+							<!-- exit menu icon -->
+							<img
+								src="/assets/icons/global/close-icon.svg"
+								alt="exit mobile menu icon"
+								class="menu-exit-icon hidden" />
 						</button>
-					</div>
 
-					<!-- Footer Navigation -->
-					<nav>
-						<ul>
+						<!-- Page Links -->
+						<ul id="nav-links" class="nav-links">
 							<li><a href="/about.html" aria-current="page">About Us</a></li>
 							<li><a href="/impact.html">Our Impact</a></li>
 							<li><a href="/values.html">Our Values</a></li>
-							<li><a href="/events.html">Events & Programs</a></li>
-							<li><a href="/contact.html">Contact</a></li>
+							<li><a href="/events.html">Events</a></li>
+							<li><a href="/team.html">Our Team</a></li>
+							<!-- Mobile Contact Button -->
+							<li class="mobile-support-btn">
+								<a href="/contact.html">Contact</a>
+							</li>
 						</ul>
-					</nav>
+					</div>
 
-					<!-- CAT Newsletter -->
-					<div class="ml-embedded" data-form="w80VC7"></div>
+					<!-- desktop support btn -->
+					<a href="/contact.html" class="btn-main desktop-support-btn">Contact Us</a>
 				</div>
+			</div>
+		</nav>
+		<!-- Hero -->
+		<section class="hero">
+			<img
+				src="/assets/images/pages/about/hero-about.jpg"
+				alt="homepage background"
+				class="img-mobile" id="hero-image" />
```

Comment:
Adding these ids (`hero-image`, `hero-heading`, `hero-subheading`) to link them to the CMS in JavaScript